Cargo.toml

```
@@ -55,11 +55,11 @@ async-std = { version = "1.0", features = ["attributes"] }
 [features]
 default = ["timing"]
 timing = ["futures-timer"]
+instrumentation = ["tracing"]
```

```suggestion
instrumentation = ["dep:tracing"]
```

This will hide the `tracing` feature which will otherwise show up [here](https://docs.rs/crate/xtra/latest/features) but not do anything.

Can be done in a follow-up PR though because we should be doing the same thing for all the others too!

---

Good point :) Let's move this to a followup as you suggested